### PR TITLE
add tool for check openvz mirrors with clair to contrib

### PR DESCRIPTION
**Pull request by mrqwer88**
Add from https://github.com/FastVPSEestiOu/check_openvz_mirror_with_clair to /contrib as told in 
Issue https://github.com/coreos/clair/issues/8

### CODE REVIEW

File: contrib/check-openvz-mirror-with-clair/main.go [hunk 1]
```diff
@@ -0,0 +1,354 @@
+package main
```
Comment (jzelinskie) [inline contrib/check-openvz-mirror-with-clair/main.go:15]: copyright headers

**Comment by mrqwer88** [comment]
Added copyright headers.

**Comment by philips** [comment]
This will need a LICENSE file in this directory if you want to put it under GPLv2. Any chance you would put it under Apache for consistency with the rest of the repo? Your call, either way is fine with me.

**Comment by mrqwer88** [comment]
I set Apache  license for consistency with rest of the repo.

**Comment by mrqwer88** [comment]
Great, thanks!

**Comment by Quentin-M** [comment]
Thanks for your contribution!

**Pull request merged**